### DISCUSSION TITLE
Multi-select filter

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#3770] [Help]
Hey, I can't use "multi-select" as a filter. No way...

Each time I end having a table with no result.

**Comment by mjashanks** [comment]
Hi @LaVoisin 

I was able to create a filter using a multi-select field type.

My component structure looks like this:

![image](https://user-images.githubusercontent.com/3524181/145792999-0a360cb3-8ffc-4c15-bced-9854ffae2642.png)

"Tags filter" is an options picker, which filters on a multi-select field.

In my `Data Provider`, under `Define Filters`, I have:

![image](https://user-images.githubusercontent.com/3524181/145793213-626cba96-11eb-45ba-a95a-9496013b60dc.png)

The table filters correctly when I select a value from my options picker.

**Reply by ghost** [reply]
Thanks you for your answer. 

In my case, both the data type and the filter are multi-select not. Maybe it's working for you because the data source is only option picker.

**Reply by mjashanks** [reply]
Yes - I actually logged this issue based on your question: https://github.com/Budibase/budibase/issues/3773 - I think that's what you need!
